frontend/src/commons/AddDropdownMenu.js

```diff
@@ -2,7 +2,7 @@ import './menu.css';
 import React, { useContext, useEffect, useRef, useState } from 'react';
 import styled from 'styled-components/macro';
 import { CgChevronRight, CgChevronLeft } from 'react-icons/cg';
-import { GiSoccerBall, GiBasketballBall } from 'react-icons/gi';
+import { GiSoccerBall } from 'react-icons/gi';
 import { BsFillCaretUpFill } from 'react-icons/bs';
 import { CSSTransition } from 'react-transition-group';
 import SoccerTeamList from '../component/SoccerTeamList';
```

Comment:
Refering to line 26: Please move to its own file and import here, makes your component smaller and you really don't have to redefine `DropdownItem` on every render ;)